service/invoker/blocks.go

```diff
@@ -0,0 +1,43 @@
+package invoker
+
```

Comment:
thanks for this! any other `/env` structures that need to be here?

Reply:
not sure, I think this is all for now.